app/uploaders/attachment_uploader.rb

```diff
@@ -1,11 +1,9 @@
 class AttachmentUploader < WhitehallUploader
   PDF_CONTENT_TYPE = "application/pdf".freeze
   INDEXABLE_TYPES = %w[csv doc docx ods odp odt pdf ppt pptx rdf rtf txt xls xlsx xml].freeze
-  EXTENSION_ALLOW_LIST = %w[chm csv diff doc docx dot dxf eps gif gml ics jpg kml odp ods odt pdf png ppt pptx ps rdf ris rtf sch txt vcf wsdl xls xlsm xlsx xlt xml xsd xslt zip].freeze
+  EXTENSION_ALLOW_LIST = %w[chm csv diff doc docx dot dxf eps gif gml ics jpg kml odp ods odt pdf png ppt pptx ps rdf ris rtf sch txt vcf wsdl xls xlsm xlsx xlt xml xsd xslt].freeze
```

Comment:
Let's also remove zip from https://github.com/alphagov/whitehall/blob/3b717cab95e965fda5583b7381d88488823e4878/app/helpers/document_helper.rb#L85

and corresponding test:
https://github.com/alphagov/whitehall/blob/3b717cab95e965fda5583b7381d88488823e4878/test/unit/app/helpers/document_helper_test.rb#L47-L49